src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -271,9 +271,6 @@ private IEnumerable<IssueMarker> RemoveSuppressedIssues(IEnumerable<IssueMarker>
             var filteredIssues = issuesFilter.Filter(filterableIssues);
             Debug.Assert(filteredIssues.All(x => x is IssueMarker), "Not expecting the issue filter to change the list item type");
 
-            var suppressedCount = filterableIssues.Count() - filteredIssues.Count();
```

Comment:
If we have an easy/elegant way to identify the issue, we could output a generic "Issue X has been suppressed"

Reply:
Even if there was, we'd see the message multiple times, in the same way we say the suppressed issue count message multiple times.

Reply:
Ah right.